pallets/evm_account_mapping/src/lib.rs

```diff
@@ -69,10 +71,25 @@ type BalanceOf<T> = <<T as Config>::Currency as Currency<<T as frame_system::Con
 type NegativeImbalanceOf<T> = <<T as Config>::Currency as Currency<<T as frame_system::Config>::AccountId>>::NegativeImbalance;
 pub type EIP712ChainID = sp_core::U256;
 pub type EIP712VerifyingContractAddress = sp_core::H160;
+pub type EIP712Signature = [u8; 65];
 
 pub type Nonce = u64;
+pub type AccountId32Bytes = [u8; 32];
+pub type EvmPublicKey = [u8; 33];
 pub type Keccak256Signature = [u8; 32];
 
+pub trait AddressConversion<T: frame_system::Config> {
+	fn convert(evm_public_key: &EvmPublicKey) -> Result<T::AccountId, ()>;
+}
+
+pub struct DefaultAddressConverter<T: frame_system::Config>(PhantomData<T>);
```

Comment:
Call it "SubstrateAddressConverter" and add a "EvmTransprentConverter"?

Reference: https://github.com/Phala-Network/phala-blockchain/pull/1495

Reply:
Renamed

Reply:
> Renamed

You might have misunderstood it. I think Hang meant:

- Renaming the struct `DefaultAddressConverter` to `SubstrateAddressConverter` rather than renaming the trait.
- Adding an additional implementation named `EvmTransparentConverter` where the formula is `let origin = evm_address ++ b"@evm_address";`.

Reply:
ok

Reply:
done